app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -7,30 +7,39 @@
 import android.view.View;
 import android.widget.EditText;
 
+import com.nervousfish.nervousfish.R;
+
 /**
  * Demo Login activity class
  */
 @SuppressWarnings("PMD")
```

Comment:
Why are you suppressing PMD? (if this is because the class isn't fully implemented yet, add a comment explaining that and at what point the suppression can be removed)